src/System.IO.Pipelines/src/System/IO/Pipelines/PipeAwaitable.cs

```diff
@@ -19,7 +19,7 @@ internal struct PipeAwaitable
         private SynchronizationContext _synchronizationContext;
         private ExecutionContext _executionContext;
 
-#if !netstandard
+#if (!netstandard && !netfx)
```

Comment:
You can use NETFRAMEWORK which is predefined by the SDK:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if

Reply:
Not on release/3.1 at least. Here are the defines I'm getting straight out of the binlog:
```
,DEBUG,TRACE,netfx;TRACE;RESOURCES_EXTENSIONS;INTERNAL_NULLABLE_ATTRIBUTES;;
```

Reply:
I believe release/3.1 is still in the place where we were not using the actual defines from the SDK yet. dotnet/runtime is different.